mmselfsup/models/heads/mae_head.py

```diff
@@ -80,3 +80,37 @@ def loss(self, outputs, labels):
         losses['loss'] = self.criterion(outputs[0], labels)
 
         return losses
+
+
+@HEADS.register_module()
+class MAELinprobeHead(BaseModule):
```

Comment:
I think maybe `MAELinearHead` is better.
Besides, it can be added in another PR.